src/tests/core/ioc_test.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+
+from src.core.ioc import Ioc
+
+
+class IocTestCase(unittest.TestCase):
+    def test_instance_not_none(self):
+
+        ioc_instance = Ioc()
+
+        product_controller_instance = ioc_instance.get_instance("product_controller")
+        self.assertIsNotNone(product_controller_instance)
+
+    def test_same_instance(self):
```

Comment:
Could you check instance 1 and 2 are also de same?

Reply:
done!